pycloudlib/ec2/instance.py

```diff
@@ -111,13 +111,18 @@ def console_log(self):
 
         """
         start = time.time()
+        output = ''
         while time.time() < start + 300:
             response = self._instance.console_output()
             try:
-                return response['Output']
+                output += response['Output']
```

Comment:
@TheRealFalcon This logic now has the effect of looping every 5 seconds for 5 minutes and adding all previous repeated full console output to the boot. The end result that you will always wait 5 mins on `ec2.console_log` call and then get about 60 copies of the full console log from the vm. Is it worth instead just adding a wait_for_cloud_init=False boolean param for cases where we need to wait cloud-init to complete before checking the logs?

Reply:
An alternative could be that we just call instance.wait() in integration tests before calling instance.console_log()

Reply:
Bah, thanks. There's something in the back of my mind thinking that we consume the log. Not sure if that's because of another cloud or I just made it up in my head.

The test will need to be updated to poll multiple times then. We are already calling `.wait()` before we start our tests, and `.restart()` also calls `.wait()`. I can back out this commit.